src/data/processors/ccCta.ts

```diff
@@ -0,0 +1,25 @@
+import { FormattedCta } from '@/types/formatted/cta'
+import { FormattingError } from '../errors/formatting'
+import type { CCFieldCta } from '@/types/drupal/paragraph'
+
+export const ccCta = (cta?: CCFieldCta[]): FormattedCta => {
+  if (!cta?.length) {
+    return null
+  }
+  const ccItem = cta[0]
+  if (
+    !ccItem?.field_button_link?.length ||
+    !ccItem?.field_button_label?.length
+  ) {
+    throw new FormattingError(
+      'cta:link or label missing: should only be used on a Cta '
+    )
+  }
+  return {
+    id: ccItem.target_id, // or pid?
```

Comment:
The question here is in centralized content, do we reference the id as the target_id or the pid (paragraph in CC). The target_id makes more sense, but can be shifted.